tree/20_cfe_basics/log_rudder.cf

```diff
@@ -142,6 +142,136 @@ bundle agent _rudder_common_reports_generic(technique_name, class_prefix, identi
         ifvarclass => "${class_prefix}_repaired";
 }
 
+
+######################################################################
+##### Reporting for the pre/post hooks
+##### These methods should not be used for normal generic method usage
+##### because they do not respect the stop on repair for audit mode
+######################################################################
```

Comment:
Could you add a WARN on standard methods to notify these should be ketp in sync?

Reply:
Done